nncf/common/quantization/quantizer_propagation/solver.py

```diff
@@ -333,6 +333,7 @@ def __init__(
         run_consistency_checks: bool = False,
         quantize_outputs: bool = False,
         post_processing_marker_metatypes: List[OperatorMetatype] = None,
+        model_type_marker_metatypes: List[OperatorMetatype] = None,
```

Comment:
Can we use more generic name that is not depending only on ModelType?
Maybe `metatypes_to_ignore`?